packages/forma-36-react-components/src/components/EntityList/EntityListItem/README.mdx

```diff
@@ -0,0 +1,30 @@
+The EntityListItem component is used to represent lists of entities (entries and assets).
```

Comment:
I would improve those docs separately, for now, is just migration to the `README` format.